atc/config.go

```diff
@@ -128,7 +128,10 @@ func (c *VersionConfig) UnmarshalJSON(version []byte) error {
 		for k, v := range actual {
 			if s, ok := v.(string); ok {
 				version[k] = strings.TrimSpace(s)
```

Comment:
I think Alex mentioned in the original issue for this PR that `TrimSpace(s)` was not needed?

Reply:
It is somehow still useful ... it will trim these
`{"some":"        value         "}` to `{"some":"value"}`

It depends on how we want to process the value string , it we say they get exactly what they put in then yeah we should remove `TrimSpace`

Reply:
@vito pls more insight

Reply:
It just seems like a weasly bit of code that never had strong justification for being there in the first place. I don't see any reason why we would want it here and not literally everywhere else in the config, lol. For example someone could say `passed: ["job-a "]` but we don't bother sanitizing that.